packages/medusa/src/api/routes/admin/invites/index.ts

```diff
@@ -32,8 +32,33 @@ export default (app) => {
   return app
 }
 
+/**
+ * @schema AdminInviteDeleteRes
+ * type: object
+ * properties:
+ *   id:
+ *     type: string
+ *     description: The ID of the deleted Invite.
+ *   object:
+ *     type: string
+ *     description: The type of the object that was deleted.
+ *     default: invite
```

Comment:
Replaced `format` with `default` for consistency's sake and is more appropriate.